x-pack/plugins/apm/public/components/app/Settings/agent_keys/agent_keys_table.tsx

```diff
@@ -144,7 +144,7 @@ export function AgentKeysTable({ agentKeys, onKeyDelete }: Props) {
         tableCaption={i18n.translate(
           'xpack.apm.settings.agentKeys.tableCaption',
           {
-            defaultMessage: 'Agent keys',
+            defaultMessage: 'APM Agent keys',
```

Comment:
```suggestion
            defaultMessage: 'APM agent keys',
```
Don't think it's capital cased (at least not in the other instances)